app/src/layouts/DashboardLayout/MenuHeader/index.js

```diff
@@ -131,35 +138,37 @@ const MenuHeader = () => {
                   />
                 </span>
               </Col> */}
-                <RQButton
-                  className="ask-ai-btn"
-                  onClick={() => {
-                    trackAskAIClicked();
-                    setIsRequestBotVisible(true);
-                    dispatch(
-                      actions.updateProductTourCompleted({
-                        tour: TOUR_TYPES.MISCELLANEOUS,
-                        subTour: "askAI",
-                      })
-                    );
-                  }}
-                  data-tour-id={MISC_TOURS.APP_ENGAGEMENT.ASK_AI}
-                >
-                  <div className="ask-ai-btn-content">
-                    <img src={BotIcon} alt="bot" />
-                    Ask AI
-                  </div>
-                </RQButton>
-
-                <Divider type="vertical" className="header-vertical-divider hidden-on-small-screen" />
-
-                {(appMode === GLOBAL_CONSTANTS.APP_MODES.DESKTOP ||
-                  (appMode !== GLOBAL_CONSTANTS.APP_MODES.DESKTOP &&
-                    user?.details?.planDetails?.status !== "canceled")) && (
-                  <Col>
-                    <PremiumPlanBadge />
-                  </Col>
-                )}
+                      <RQButton
+                        className="ask-ai-btn"
+                        onClick={() => {
+                          trackAskAIClicked();
+                          setIsRequestBotVisible(true);
+                          dispatch(
+                            actions.updateProductTourCompleted({
+                              tour: TOUR_TYPES.MISCELLANEOUS,
+                              subTour: "askAI",
+                            })
+                          );
+                        }}
+                        data-tour-id={MISC_TOURS.APP_ENGAGEMENT.ASK_AI}
+                      >
+                        <div className="ask-ai-btn-content">
+                          <img src={BotIcon} alt="bot" />
+                          Ask AI
+                        </div>
+                      </RQButton>
+                    </>
+                  )}
+                  <Divider type="vertical" className="header-vertical-divider hidden-on-small-screen" />
+                  {(appMode === GLOBAL_CONSTANTS.APP_MODES.DESKTOP ||
+                    (appMode !== GLOBAL_CONSTANTS.APP_MODES.DESKTOP &&
+                      user?.details?.planDetails?.status !== "canceled")) &&
+                    isAppTypeRequestly && (
```

Comment:
this check won't be required if constant is moved to [common/constants.js under CONSTANTS.APP_MODES](https://github.com/requestly/requestly/blob/6db0cbf3eb80ea230a530912e54d514122e44b5d/common/constants.js#L10-L16)